blocks/page_list/templates/itinerary/view.less

```diff
@@ -0,0 +1,24 @@
+.itinerary-walk {
+  max-width: 70%;
+  float: left;
+}
+
+.itinerary-remove-walk {
```

Comment:
I don't like seeing such verbose class naming around this - it reduces our ability to generalize styles, and semantically it doesn't make much sense. e.g. why is this element an "itinerary remove walk"? It already appears in the tree for 'itinerary', so it's redundant to call it something beginning with `.itinerary`. We may define other lists later which aren't itineraries, but follow the same general style and design. You would extend them,  e.g. `.walkReviews { &:extend(.itinerary all)`